omop_file_validator.py

```diff
@@ -67,16 +69,18 @@ def cast_type(cdm_column_type, value):
     :param value:
     :return:
     """
-    if cdm_column_type == 'integer':
+    if cdm_column_type in ('integer', 'int64'):
         return int(value)
     if cdm_column_type in ('character varying', 'text', 'string'):
         return str(value)
     if cdm_column_type == 'numeric':
         return float(value)
-    if cdm_column_type == 'date':
+    if cdm_column_type == 'float' and isinstance(value, float):
+        return value
+    if cdm_column_type == 'date' and isinstance(value, datetime.date):
         return datetime.date(value)
```

Comment:
If it is an instance of `datetime.date`, should it be converted to `datetime.date` again?

Reply:
this is a mistake. it should just return the value in this instance.

Reply:
Hey @noahgengel, once this is fixed I'll go ahead and merge it.